package_manifest.yaml

```diff
@@ -78,7 +78,7 @@ copr_projects:
           comps_file: "{{ inventory_dir }}/comps/comps-foreman-plugins-el{{ rhel_8 }}.xml"
           buildroot_packages: "{{ core_buildroot_packages }}"
     katello-copr:
-      copr_project_name: "katello-{{ foreman_version }}-staging"
+      copr_project_name: "katello-{{ katello_version }}-staging"
```

Comment:
@ehelms this should be Katellote_version and not Foreman-version right?

Reply:
Good catch, yes it should.